pom.xml

```diff
@@ -108,7 +108,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.27</version>
+            <version>1.33</version>
```

Comment:
ditto, plus some follow-up fixes likely to affect testing of Groovy-heavy code like this

Reply:
We could theoretically go straight to 1.34, but 1.33 is good enough, yeah.